src/Microsoft.TestPlatform.Utilities/CodeCoverageRunSettingsProcessor.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.Utilities
+{
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel;
+    using System;
+    using System.Collections.Generic;
+    using System.Xml;
+    using System.Xml.XPath;
+
+    public class CodeCoverageRunSettingsProcessor
+    {
+        #region Members
+        private static readonly string CodeCoverageCollectorDefaultSettings =
+            @"<DataCollector uri=""datacollector://microsoft/CodeCoverage/2.0"" assemblyQualifiedName=""Microsoft.VisualStudio.Coverage.DynamicCoverageDataCollector, Microsoft.VisualStudio.TraceCollector, Version=16.0.0.0 " + @", Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a"" friendlyName=""Code Coverage"">" + Environment.NewLine +
```

Comment:
Make it a single multiline string? (or if you decide to concatenate then string array and string.join by environment.newline?)

Reply:
I copy-pasted the string as is from `EnableCodeCoverageArgumentProcessor.cs` and I prefer to keep it formatted like this to give the reader a better understanding of the structure of the XML. However, I decided to go with your second option, i.e. `string.Join` on a string array due to perf considerations (also it's easier than writing by hand the concatenation of `Environment.NewLine` every time :) ). Even though the string has a constant number of concatenations and the operation is only done once because it's a static it should still be faster this way 👍